presto-product-tests/conf/docker/common/compose-commons.sh

```diff
@@ -12,6 +13,7 @@ function canonical_path() {
         echo "Invalid path: [$PATH]. Working directory is [`pwd`]."
```

Comment:
I propose that this should be echo'ed to >&2 to send it to stderr rather than stdout.